server/isomorphic-routing/index.js

```diff
@@ -66,7 +66,6 @@ function combineMiddlewares( ...middlewares ) {
 // TODO: Maybe merge into getDefaultContext().
 function getEnhancedContext( req, res ) {
 	return Object.assign( {}, req.context, {
-		isLoggedIn: req.cookies.wordpress_logged_in,
```

Comment:
`isLoggedIn` is now defined in `getDefaultContext()` here:
https://github.com/Automattic/wp-calypso/pull/30157/files#diff-b2d6784517d1151d7e75f0287d0c6478R274